packages/pds/src/db/message-queue/index.ts

```diff
@@ -0,0 +1,276 @@
+import { RepoStructure } from '@atproto/repo'
+import Database from '..'
+import {
+  AddMember,
+  AddUpvote,
+  CreateNotification,
+  DeleteNotifications,
+  Message,
+  RemoveMember,
+  RemoveUpvote,
+} from './messages'
+import { sql } from 'kysely'
+import { dbLogger as log } from '../../logger'
+import { SceneVotesOnPost } from './tables/sceneVotesOnPost'
+import { AuthStore } from '@atproto/auth'
+import * as repoUtil from '../../util/repo'
+import * as schema from '../../lexicon/schemas'
+import { TID } from '@atproto/common'
+import { MessageQueue } from '../types'
+
+export class SqlMessageQueue implements MessageQueue {
```

Comment:
I'm being a jerk by not checking myself, but just make sure you're using Kafka's semantics here, both for API and the expected fields. I'd suggest copying the exact fields of messages in Kafka, even if they don't apply to the table system, just so we don't get caught up with awkward abstractions over the differences in the future